ova/generate_ova.sh

```diff
@@ -55,163 +64,191 @@ clean() {
 
+    echo "Exporting ova"
+
+    # Get machine name
+    VM_EXPORT=$(vboxmanage list vms | grep -i vm_wazuh | cut -d "\"" -f2)
+    
+    # Create OVA with machine
+    vboxmanage export ${VM_EXPORT} -o ${OVA_VM} \
+    --vsys 0 \
+    --product "Wazuh v${WAZUH_VERSION} OVA" \
+    --producturl "https://packages.wazuh.com/vm/wazuh-${OVA_VERSION}.ova" \
+    --vendor "Wazuh, inc <info@wazuh.com>" --vendorurl "https://wazuh.com" \
+    --version "$OVA_VERSION" --description "Wazuh helps you to gain security visibility into your infrastructure by monitoring hosts at an operating system and application level. It provides the following capabilities: log analysis, file integrity monitoring, intrusions detection and policy and compliance monitoring." \
+    || clean 1
+
+    # Destroy vagrant machine
     vagrant destroy -f
+
+    # Extract ova
     tar -xvf ${OVA_VM}
 
+    echo "Setting up ova for VMware ESXi"
+
+    # Configure OVA for import to VMWare ESXi
     python Ova2Ovf.py -s ${OVA_VM} -d ${OVA_FIXED}
 
+    # Make output dir of OVA file
     mkdir -p ${OUTPUT_DIR}
     mv ${OVA_FIXED} ${OUTPUT_DIR}/${OVA_VM}
 
+    # Check Checksum
     if [ "${CHECKSUM}" = "yes" ]; then
         mkdir -p ${CHECKSUM_DIR}
         cd ${OUTPUT_DIR} && sha512sum "${OVA_VM}" > "${CHECKSUM_DIR}/${OVA_VM}.sha512"
     fi
 
-    # Cleaning tasks
-    clean 0
 }
 
 check_version() {
-    if [ "${PACKAGES_REPOSITORY}" = "prod" ]; then
-        WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
-        if [ "${WAZUH_MAJOR}" = "3" ]; then
-            curl -Isf https://packages.wazuh.com/${WAZUH_MAJOR}/ui/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION} not supported." && exit 1 )
+    WAZUH_MAJOR="$(echo ${WAZUH_VERSION} | head -c 1)"
+    if [ "${WAZUH_MAJOR}" = "4" ]; then
+        if [ "${PACKAGES_REPOSITORY}" = "prod" ]; then
+            curl -Isf https://packages.wazuh.com/${WAZUH_MAJOR}.x/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-${UI_REVISION}.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION}-${UI_REVISION} not supported." && exit 1 )
+        elif [ "${PACKAGES_REPOSITORY}" = "dev" ]; then
+            curl -Isf https://packages-dev.wazuh.com/pre-release/ui/kibana/wazuh_kibana-${WAZUH_VERSION}_${ELK_VERSION}-${UI_REVISION}.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION} not supported." && exit 1 )
         else
-            curl -Isf https://packages.wazuh.com/${WAZUH_MAJOR}/ui/kibana/wazuhapp-${WAZUH_VERSION}_${ELK_VERSION}-${UI_REVISION}.zip > /dev/null || ( echo "Error version ${WAZUH_VERSION}-${ELK_VERSION}-${UI_REVISION} not supported." && exit 1 )
+            logger "Error, repository value must take 'prod' (production) or 'dev' (development) value."
```

Comment:
Where is the logger function? I think you removed it. You can remove logger occurrences OR include the logger function in this file. Use logger OR echo, not both.

Reply:
There was no logger function in this file, the reasons for use logger and echo are:

- logger stores information in the system logs, it can be consulted with journalctl and the log file depends on the system, usually it is /var/log/syslog

- echo for the terminal output

Reply:
Fixed in https://github.com/wazuh/wazuh-packages/pull/746/commits/47c6226df8ec50fffa25781917902d4492cff272